docs/docusaurus/docs/components/examples_under_test.py

```diff
@@ -48,5 +53,62 @@
     # ),
 ]
 
+expectation_tests = [
```

Comment:
Since this is all commented out right now, I'd add a comment to the top of this that explains why, eg: `TODO: As we get these tests working, uncomment to add these to CI.`